doc/move_group_interface/src/move_group_interface_tutorial.cpp

```diff
@@ -456,11 +425,16 @@ int main(int argc, char** argv)
   // END_SUB_TUTORIAL
 
   // Plan
-  success = move_group_interface_tutorial.planJointSpaceGoal(joint_group_positions, plan);
-  move_group_interface_tutorial.publishToRViz("Joint Space Goal", plan);
+
+  publish_message = "Joint Space Goal";
+  // Create the plan we will be using
+  moveit::planning_interface::MoveGroupInterface::Plan plan;
+  bool success = move_group_interface_tutorial.planJointSpaceGoal(joint_group_positions, plan);
+  move_group_interface_tutorial.publishToRViz(publish_message, &plan);
   move_group_interface_tutorial.move(plan);
 
-  move_group_interface_tutorial.publishToRViz("Pose Goal");
+  publish_message = "Pose Goal";
```

Comment:
let's keep minor changes like this to a minimum and focus on getting this merged